### PR TITLE
add related work in readme

### PR DESCRIPTION
**Pull request by Yongtae723**
As talking #178, We create a repository of PyTorch Lightning implementations of SetFit.

If you think this repository is worth an introduction, I would be happy to merge this PR.

Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -380,6 +380,8 @@ make style && make quality
 └── tests           <- Unit tests
 ```
 
+## Related work
+[jxpress/seifit-pytorch-lightning](https://github.com/jxpress/seifit-pytorch-lightning) - A repository of PyTorch Lightning implementations of SetFit.
```
Comment (lewtun) [inline README.md:384]: small suggestion to fix a typo and create a bullet list :)
```suggestion

* [jxpress/setfit-pytorch-lightning](https://github.com/jxpress/setfit-pytorch-lightning) - A PyTorch Lightning implementation of SetFit.
```

**Review by lewtun — APPROVED** [review]
Cool work @Yongtae723 on the PyTorch Lightning implementation! Happy to include a reference to it and I've left a small nit before we can merge

File: README.md (hunk 1 above)
Comment (Yongtae723) [reply README.md]: Hi @lewtun 
thank you for approving my PR and suggestion!

I fixed it as you suggested me.

thank you for kind reply :)

Yongtae

**Comment by lewtun** [comment]
Thanks for iterating!

**Pull request merged**